### DISCUSSION TITLE
How can I embed a private vimeo video

### DISCUSSION THREAD

**Discussion by willLopesz** [MetaLabs-inc/react-native-vimeo-iframe#40] [General]
How can I embed a private vimeo video using this librarie?

**Comment by Marcoo09** [comment]
Hi @willLopesz, on the new release it was included this feature.

**Comment by matiassalles99** [comment]
@Marcoo09 is this included already?

**Comment by Marcoo09** [comment]
@willLopesz @matiassalles99 
It should be used the `h`parameter as explained [here](https://developer.vimeo.com/player/sdk/basics#connecting-to-a-vimeo-player)
The video in vimeo should be setted as **Hidden in Vimeo**
